code/game/objects/items/stacks/stack_recipe.dm

```diff
@@ -28,8 +28,84 @@
 	src.window_checks = window_checks
 	src.cult_structure = cult_structure
 
-/datum/stack_recipe/proc/post_build(obj/item/stack/S, obj/result)
-	return
+///Returns TRUE if the recipe can be built, otherwise returns FALSE.  This proc is only meant as a series of tests to check if construction is possible; the actual creation of the resulting atom should be handled in do_build()
```

Comment:
```suggestion
/// Returns TRUE if the recipe can be built, otherwise returns FALSE. This proc is only meant as a series of tests to check if construction is possible; the actual creation of the resulting atom should be handled in do_build()
```